src/library/scala/collection/IterableOnce.scala

```diff
@@ -315,9 +321,10 @@ object IterableOnce {
   * @define consumesIterator
   *              After calling this method, one should discard the iterator it was called
   * on. Using it is undefined and subject to change.
-  * @define undefinedorder
-  *              The order in which operations are performed on elements is unspecified
-  *              and may be nondeterministic.
+  * @define undefinedOrder
+  *              The order of applications of the operator is unspecified and may be nondeterministic.
+  * @define exactlyOnce
+  *              Each element appears exactly once in the computation.
```

Comment:
maybe "is used" rather than "appears", wdyt?

Reply:
"appears" is "as written"; Lukas suggested it recently for a user expression.

Reply:
This is a very minor point and I don’t feel strongly about it. But “is used” to me suggests that the element is being consumed/‘used up’ in some way. Plus, the operator may ‘use’ the element multiple times internally, but when describing the whole fold operation in terms of an abstract operator, the element is guaranteed to ‘appear’ exactly once.

Reply:
Yes, I agree that in support of "appear", we mean the operation written out as an abstraction.

One may ask whether the "computation" is the expression of the program or the formula to which it is supposed to correspond, but I won't ask that.